metricbeat/module/kubernetes/util/metrics_cache.go

```diff
@@ -0,0 +1,106 @@
+package util
+
+import (
+	"sync"
+	"time"
+)
+
+// PerfMetrics stores known metrics from Kubernetes nodes and containers
+var PerfMetrics *PerfMetricsCache
```

Comment:
How about init it directly here
`var PerfMetrics = NewPerfMetricsCache()`

Reply:
Sounds good to me, pushed the change, thanks!